src/data/CommandTree1.20.3.ts

```diff
@@ -0,0 +1,3667 @@
+                                        },
+                                        pos: {
+                                            parser: new VectorArgumentParser(3),
+                                            executable: true
+                                        }
+                                    }
+                                },
+                                rotation: {
+                                    parser: new VectorArgumentParser(2, 'float', false),
+                                    executable: true
+                                }
+                            }
+                        },
+                        entity: {
+                            parser: new EntityArgumentParser('single', 'entities'),
+                            executable: true
+                        }
+                    }
+                }
+            }
+        },
+        tell: {
+            redirect: 'commands.msg'
+        },
+        tellraw: {
+            parser: new LiteralArgumentParser('tellraw'),
+            children: {
+                player: {
+                    parser: new EntityArgumentParser('multiple', 'players'),
+                    children: {
+                        message: {
+                            parser: new TextComponentArgumentParser(),
+                            executable: true
+                        }
+                    }
+                }
+            }
+        },
+        tick: {
+            parser: new LiteralArgumentParser('tick'),
+            children: {
+                [Switchable]: true,
+                freeze: {
+                    parser: new LiteralArgumentParser('freeze'),
+                    executable: true
+                },
+                query: {
+                    parser: new LiteralArgumentParser('query'),
+                    executable: true
+                },
+                rate: {
+                    parser: new LiteralArgumentParser('rate'),
+                    children: {
+                        rate: {
+                            parser: new NumberArgumentParser('float', 1, 10_000),
+                            executable: true
+                        }
+                    }
+                },
+                sprint: {
+                    parser: new LiteralArgumentParser('sprint'),
+                    children: {
+                        [Switchable]: true,
+                        stop: {
+                            parser: new LiteralArgumentParser('stop'),
+                            executable: true
+                        },
+                        time: {
+                            parser: new TimeArgumentParser(),
+                            executable: true
+                        }
+                    }
+                },
+                step: {
+                    parser: new LiteralArgumentParser('sprint'),
+                    children: {
+                        [Switchable]: true,
```

Comment:
Same as sprint.
and forgot to change.
```suggestion
                step: {
                    parser: new LiteralArgumentParser('step'),
                    children: {
```